.travis.yml

```diff
@@ -1,6 +1,5 @@
 language: python
 python:
-  - "2.6"
```

Comment:
Looks like I forgot to update `.travis.yml` when I re-added support for it. I'll fix that.